package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "watchdog-proxy",
+  "version": "0.0.1",
+  "description": "Rate limiting proxy for watchdog requests",
+  "main": "index.js",
```

Comment:
Nit: probably remove this, as I don't think we have an ./index.js file (and this is probably just the npm default).